frontend/packages/component-library/src/cms/iframeEmbedContainer/IframeEmbedContainer.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import {IframeHTMLAttributes} from 'react';
+
+import moduleStyles from './iframeEmbedContainer.module.scss';
+
+export interface IframeEmbedContainerProps
+  extends IframeHTMLAttributes<HTMLIFrameElement> {
+  /** URL of the embedded content */
+  src: string;
+  /** Title */
+  title: string;
+  /** Class of the embed container */
+  className?: string;
+  /** Height */
+  height?: number | string;
+  /** Width */
+  width?: number | string;
```

Comment:
I wonder if we should add these two as a prop?
It looks to me that if we plan to set height and width to the Iframe/iframe container, we can either do it via className usage or via contentful editor. iframe element in this case can always have `100%` for height and width.

Reason I'm asking this - as a rule, if possible, we try not to use inline styles in component-library

Reply:
Can we leave this as-is for now until we see how it works in Contentful?